src/crimson/net/Connection.h

```diff
@@ -40,6 +45,20 @@ class Connection : public seastar::enable_shared_from_this<Connection> {
  public:
   uint64_t peer_global_id = 0;
 
+ protected:
+  uint64_t features;
```

Comment:
shall we initialize it with a known value?

Reply:
The constructor of our `SocketConnection` doesn't touch it while the classical `Connection` zero-initializes the member:

```cpp
  Connection(CephContext *cct, Messenger *m)
    // we are managed exclusively by ConnectionRef; make it so you can
    //   ConnectionRef foo = new Connection;
    : RefCountedObject(cct, 0),
      // ...
      features(0),
      // ...
```

Let's use this opportunity to make a small clean-up.